sphinx/source/docs/user_guide/tools.rst

```diff
@@ -71,7 +71,7 @@ outside of the area where the axis is drawn.
 
 .. _userguide_tools_specifying_tools:
 
-Specifying Tools
+Specifying tools
 ----------------
 
 At the lowest ``bokeh.models`` level, tools are added to a ``Plot`` by
```

Comment:
Generally, I try to include the brackets "()" when talking about a function or method (as opposed to an argument or property, for example). So I'd say it should be "``add_tools()`` method" in this section (instead of ``add_tools``). I'm aware that we are not really consistent with this throughout the documentation, and it seem that in this section, we had not included the brackets in the past.
@bryevdv, what do you think?